test/server/cards/02_SHD/leaders/HondoOhnakaThatsGoodBusiness.spec.ts

```diff
@@ -0,0 +1,136 @@
+describe('Hondo Ohnaka, That\'s Good Business', function () {
+    integration(function (contextRef) {
+        describe('Hondo Ohnaka\'s leader ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['green-squadron-awing'],
+                        leader: 'hondo-ohnaka#thats-good-business',
+                    },
+                    player2: {},
+                });
+            });
+
+            it('should not give experience token when play a unit from hand', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.greenSquadronAwing);
+                expect(context.player2).toBeActivePlayer();
+            });
+        });
+
+        describe('Hondo Ohnaka\'s leader ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        resources: ['privateer-crew', 'warbird-stowaway', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst'],
+                        groundArena: ['battlefield-marine'],
+                        leader: 'hondo-ohnaka#thats-good-business',
+                    },
+                    player2: {
+                        spaceArena: ['green-squadron-awing'],
+                        resources: ['freetown-backup', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst', 'atst']
+                    },
+                });
+            });
+
+            it('should give experience token when play a unit from smuggle', function () {
+                const { context } = contextRef;
+
+                // play a unit from smuggle
+                context.player1.clickCard(context.privateerCrew);
+
+                // choose between 2 triggers
+                context.player1.clickPrompt('Exhaust this leader');
+                expect(context.player1).toHavePassAbilityPrompt('Exhaust this leader');
+                context.player1.clickPrompt('Exhaust this leader');
+
+                // add experience to battlefield marine
+                expect(context.player1).toBeAbleToSelectExactly([context.battlefieldMarine, context.privateerCrew, context.greenSquadronAwing]);
+                context.player1.clickCard(context.battlefieldMarine);
+                expect(context.player2).toBeActivePlayer();
+                expect(context.battlefieldMarine).toHaveExactUpgradeNames(['experience']);
+            });
+
+            it('should give experience token when play a unit from smuggle (pass on first smuggle played)', function () {
```

Comment:
I think this test case makes the one above not needed? We could just bring in the detailed checks from that one and remove it. That's the way we've handled it in other cases.